src/sagemaker/jumpstart/utils.py

```diff
@@ -982,3 +984,38 @@ def get_jumpstart_configs(
         if metadata_configs
         else {}
     )
+
+
+def get_jumpstart_user_agent_extra_suffix(model_id: str, model_version: str) -> str:
+    """Returns the model-specific user agent string to be added to requests."""
+    sagemaker_python_sdk_headers = get_user_agent_extra_suffix()
+    jumpstart_specific_suffix = f"js_model_id/{model_id} js_model_version/{model_version}"
```

Comment:
Can we make it a constant? Something like JUMPSTART_UA_SUFFIX = `md/js_model_id#<id> js_model_ver#<version>` This follows the guidelines of UserAgent string as shared in primate chat.